templates/field/field--node--oe-person-interests-intro--oe-person--full.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Declaration of interests introduction field template .
+ *
+ * @see ./core/themes/stable/templates/field/field.html.twig
+ */
+#}
+{% for item in items %}
+  <div class="ecl-u-mb-l ecl-editor">
```

Comment:
Why is this large but the introduction to the biography its medium?

Reply:
Because of design
<img width="1154" alt="Screenshot 2021-03-10 at 20 51 11" src="https://user-images.githubusercontent.com/1183951/110673940-70344d80-81e2-11eb-817a-752420d6ee87.png">